testing/py-unittest2/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+pkgname=py-unittest2
+_pkgname=${pkgname#py-*}
+pkgver=1.1.0
+pkgrel=0
+pkgdesc="Backport of the Python3 enhanced unittest testing framework"
+url="https://pypi.python.org/pypi/unittest2"
+arch="noarch"
+license="PSF"
+depends="python py-six py-argparse py-traceback2 py-linecache2"
```

Comment:
Please don’t declare versionless python packages as dependencies, it should be python2, py2-six, py2-argparse, …